apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/SvFrontendIntegrationTest.scala

```diff
@@ -139,26 +143,36 @@ class SvFrontendIntegrationTest
           },
         )
 
-        val licenseRows = getLicensesTableRows
-        val newValidatorParty = allocateRandomSvParty("splice-client", Some(2))
+        // The ReceiveFaucetCouponTrigger, via RecordValidatorLivenessActivity, continuously
+        // recreates the existing ValidatorLicenses. This interferes with the following
+        // test due to its short tick duration, so we pause the trigger for this check.
+        TriggerTestUtil.setTriggersWithin(triggersToPauseAtStart =
+          Seq(
+            sv1ValidatorBackend.validatorAutomation
+              .trigger[ReceiveFaucetCouponTrigger]
+          )
+        ) {
+          val licenseRows = getLicensesTableRows
+          val newValidatorParty = allocateRandomSvParty("splice-client", Some(2))
 
-        actAndCheck(
-          "onboard new validator using the secret",
-          sv1Backend.onboardValidator(
-            newValidatorParty,
-            newSecret,
-            s"${newValidatorParty.uid.identifier}@example.com",
-          ),
-        )(
-          "a new validator row is added",
-          _ => {
-            checkLastValidatorLicenseRow(
-              licenseRows.size.toLong,
-              sv1Backend.getDsoInfo().svParty,
+          actAndCheck(
+            "onboard new validator using the secret",
+            sv1Backend.onboardValidator(
               newValidatorParty,
-            )
-          },
-        )
+              newSecret,
+              s"${newValidatorParty.uid.identifier}@example.com",
+            ),
+          )(
+            "a new validator row is added",
+            _ => {
+              checkLastValidatorLicenseRow(
```

Comment:
So we seem to be sorting on the contract  created at, I suggest we just remove the check last validator license row and instead just check the license is in the list.
Furthermore i would honestly just remove this test all together and move it to vite

Reply:
>  one minimal selenium test for real api calls from the UI for each page.

:+1: